source/ed/build_helper.h

```diff
@@ -51,26 +51,64 @@ struct builder;
 
 /// Structure retournée à la construction d'un VehicleJourney
 struct VJ {
-    builder & b;
-    nt::VehicleJourney* vj;
+    builder& b;
+    const std::string network_name;
+    const std::string line_name;
+    const std::string validity_pattern;
+    const std::string block_id;
+    const bool is_frequency;
+    const bool wheelchair_boarding;
+    const std::string uri;
+    const std::string meta_vj_name;
+    const std::string physical_mode;
+    const uint32_t start_time;
+    const uint32_t end_time;
+    const uint32_t headway_secs;
+    std::vector<nt::StopTime> sts;
+    nt::VehicleJourney* vj = nullptr;
 
     /// Construit un nouveau vehicle journey
-    VJ(builder & b, const std::string &line_name, const std::string &validity_pattern,
-       bool is_frequency,
-       bool wheelchair_boarding = true, const std::string& uri="",
-       const std::string& meta_vj_name = "", const std::string& physical_mode = "");
+    VJ(builder& b,
+       const std::string& network_name,
+       const std::string& line_name,
+       const std::string& validity_pattern,
+       const std::string& block_id,
+       const bool is_frequency,
+       const bool wheelchair_boarding = true,
+       const std::string& uri="",
+       const std::string& meta_vj_name = "",
+       const std::string& physical_mode = "",
+       const uint32_t start_time = 0,
+       const uint32_t end_time = 0,
+       const uint32_t headway_secs = 0);
+
+    VJ(VJ&&) = default;
+    VJ& operator=(VJ&&) = delete;
+    VJ(const VJ&) = delete;
+    VJ& operator=(const VJ&) = delete;
+    ~VJ() { make(); }
```

Comment:
Yes, the destructor create the vj, and you can create manually the vj by calling make() by hand.